db/schema.rb

```diff
@@ -60,11 +60,6 @@
     t.index ["updated_at"], name: "index_events_on_updated_at"
   end
 
-  create_table "events_tags", id: false, force: :cascade do |t|
-    t.bigint "event_id", null: false
-    t.bigint "tag_id", null: false
-  end
```

Comment:
think this was auto-generated from `rails g migration no_empty_tags description:string` not sure why though. This table isn't used anywhere in the past anyway. we use `event_tags` for event-tag relationship.